tests/auto/make_dump.py

```diff
@@ -301,18 +309,18 @@ def read_operand(o):
         #There is a capstone branch that should fix all this annoyances .. soon
         #https://github.com/aquynh/capstone/tree/next
         used = set()
-        for ri in reg_sizes.keys():
+        for ri in list(reg_sizes.keys()):
             if instruction.reg_read(ri) or instruction.reg_write(ri):
                 if not(instruction.reg_read(reg_sizes[ri]) or instruction.reg_write(reg_sizes[ri])):
-                    if str(instruction.reg_name(reg_sizes[ri]).upper()) not in registers.keys():
+                    if str(instruction.reg_name(reg_sizes[ri]).upper()) not in list(registers.keys()):
```

Comment:
It can be just `if str(...) not in registers:`

See:
```
In [1]: a = {1:2, 3:4, 5:6}

In [2]: if 1 in a:
   ...:     print("OK")
   ...:     
OK
```